src/main/kotlin/app/revanced/patches/twitter/layout/hideviews/fingerprints/TweetStatsContainerConstructorFingerprint.kt

```diff
@@ -0,0 +1,22 @@
+package app.revanced.patches.twitter.layout.hideviews.fingerprints
+
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import org.jf.dexlib2.AccessFlags
+import org.jf.dexlib2.Opcode
+
+object TweetStatsContainerConstructorFingerprint : MethodFingerprint(
+    access = AccessFlags.PUBLIC or AccessFlags.CONSTRUCTOR,
+    parameters = listOf("Landroid/view/View;"),
```

Comment:
For parameter types only the prefixing char is checked:

```suggestion
    parameters = listOf("L"),
```